docs/ops/generation/Multinomial_13.md

```diff
@@ -0,0 +1,181 @@
+# Multinomial {#openvino_docs_ops_generation_Multinomial_13}
+
+@sphinxdirective
+
+.. meta::
+  :description: Learn about Multinomial-13 - a generation operation, that creates a sequence of indices of sampled classes from Multinomial distribution.
+
+**Versioned name**: *Multinomial-13*
+
+**Category**: *Generation*
+
+**Short description**: *Multinomial* operation generates a sequence of class indices sampled from Multinomial distribution.
+
+**Detailed description**: *Multinomial* operation generates a sequence of class indices sampled from a Multinomial distribution. In this context, the *input* values represent the probabilities associated with each class within the Multinomial distribution. When performing the operation, it randomly selects a class based on these probabilities. Subsequently, the index of the chosen class in the *input* array is appended to the *output* sequence in the corresponding batch.
+
+**Algorithm formulation**:
+
+Given a list of probabilities x1,x2, ..., xn:
+* If *log_probs* is true:
+  * For each probability x, replace it with a value e^x
+* Create a cumulative sum of those probabilities, ie. create an array of values where the ith value is the sum of the probabilities x1, ..., xi
+* Divide the created array by its last (maximum) value to normalize the cumulative probabilities between [0, 1]
+* Randomly generate a sequence of floats in the range range [0,1]
```

Comment:
```suggestion
* Randomly generate a sequence of floats in the range range [0, 1]
```
Please clarify if this range must be a `float` number or can be any other type like `double`, or some `fixed-point`.
Maybe it should be mention here that it has to be any real number between [0, 1]? 
Consider to use some mathematical notation to explain the range value.

Reply:
Clarified to `double` precision. Added mathematical notation  [a, b] => {x ~ R, a<=x<=b}